src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -27,9 +27,8 @@ tags:
 {% Aside 'caution' %} Declarative Link Capturing was part of the
 [capabilities project](/fugu-status/).
 The engineering team has decided that Declarative Link Capturing will _not_
-launch with the current design. Interested parties
-can check out and provide early feedback on the likely replacement feature called
-[Web App Launch Handling](https://github.com/WICG/sw-launch/blob/main/launch_handler.md).
+launch in the current shape. Instead, the feature has been redesigned as
```

Comment:
```suggestion
launch with its current design. Instead, the feature has been redesigned as
```

Reply:
@tomayac Apologies. I didn't release this was a PTAL. I started reviewing as though it were new and found this. We should probably fix it.